index.js

```diff
@@ -1,13 +1,44 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort((a, b) => a - b);
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos <= this.length) {
+      return this.items[pos];
+      
+    } else if (pos >= this.length) {
+        throw new Error('OutofBounds');
+      }
+    
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error('EmptySortedList');
 
-  min() {}
+    } else if (this.length != 0) {
+        return this.items.reduce((max, num) => (max > num)? max:num);
```

Comment:
Esto esta genial!!! un par de detalles, aqui en JS necesitan respirar las variables asi que si le metes un aire entre lo que tiene que evaluar el ternario y lo que devuelve pues genial! te pongo un ejemplo
```suggestion
        return this.items.reduce((max, num) => (max > num) ? max : num);
```

Y luego se que te funciona pero siempre que uses un reduce usa o especifica mas bien el valor predeterminado que tiene max, ahora te funciona porque JS es asi de guay pero es de buenas practicas usarlo bien, en la primera vuelta el valor predeterminado de max seria undefined segun tu codigo, y cuando le sumas o lo evaluas con un numero siempre te dira que el numero que evaluas pues es mayor o lo que sea, para evitar esto por si acaso, dale un valor predeterminado, despues de esta turra te pongo lo que haria 
```suggestion
        return this.items.reduce((max, num) => (max > num) ? max : num, 0);
```

con eso queda mucho mejor